app/lib/features/chat/pages/room_page.dart

```diff
@@ -73,193 +75,220 @@ class _ChatRoomConsumerState extends ConsumerState<ChatRoom> {
+                  ),
+                  const SizedBox(height: 5),
+                  activeMembers.when(
+                    data: (members) {
+                      int count = members.length;
+                      return Text(
+                        '$count ${AppLocalizations.of(context)!.members}',
+                        style: Theme.of(context).textTheme.bodySmall,
                       );
                     },
+                    skipLoadingOnReload: false,
+                    error: (error, stackTrace) =>
+                        Text('Error loading members count $error'),
                     loading: () => const CircularProgressIndicator(),
                   ),
+                ],
+              ),
+              actions: [
+                GestureDetector(
+                  onTap: () {
+                    inSideBar
+                        ? ref
+                            .read(hasExpandedPanel.notifier)
+                            .update((state) => true)
+                        : context.pushNamed(
+                            Routes.chatProfile.name,
+                            pathParameters: {'roomId': convo.getRoomIdStr()},
+                          );
+                  },
+                  child: Padding(
+                    padding: const EdgeInsets.only(right: 10),
+                    child: SpaceParentBadge(
+                      roomId: convo.getRoomIdStr(),
+                      badgeSize: 20,
+                      child: convoProfile.when(
+                        data: (profile) => ActerAvatar(
+                          uniqueId: convo.getRoomIdStr(),
+                          mode: profile.isDm
+                              ? DisplayMode.User
+                              : DisplayMode.Space,
+                          displayName:
+                              profile.displayName ?? convo.getRoomIdStr(),
+                          avatar: profile.getAvatarImage(),
+                          size: 36,
+                        ),
+                        skipLoadingOnReload: true,
+                        error: (err, stackTrace) {
+                          debugPrint('Failed to load avatar due to $err');
+                          return ActerAvatar(
+                            uniqueId: convo.getRoomIdStr(),
+                            mode: DisplayMode.GroupChat,
+                            displayName: convo.getRoomIdStr(),
+                            size: 36,
+                          );
+                        },
+                        loading: () => const CircularProgressIndicator(),
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            SliverFillRemaining(
+              child: Container(
+                decoration: const BoxDecoration(
+                  gradient: AppTheme.primaryGradient,
+                ),
+                child: Chat(
```

Comment:
I assume all that is only because of indentation changes...

Reply:
Precisely.